GuildedRoseLLC/Details/DetailsViewController.swift

```diff
@@ -13,5 +15,16 @@ public class DetailViewController: UIViewController {
         }
         
         itemName.text = item?.name
+        sellIn.text = item?.sellIn ??? "None"
+        quality.text = item?.quality ??? "None"
+    }
+}
+
+infix operator ???: NilCoalescingPrecedence
```

Comment:
I'm a bit confused by what's going on here. What is `NilCoalescingPrecedence`? And more generally, what is this code doing?